backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -658,27 +671,39 @@ public Optional<OrganizationRoleClaims> getOrganizationRoleClaimsForUser(String
         getUserOrThrowError(username, "Cannot get org external ID for nonexistent user"));
   }
 
-  public Integer getUsersInSingleFacility(Facility facility) {
-    String facilityAccessGroupName =
-        generateFacilityGroupName(
-            facility.getOrganization().getExternalId(), facility.getInternalId());
-
-    List<Group> facilityAccessGroup =
-        groupApi.listGroups(facilityAccessGroupName, null, null, 1, "stats", null, null, null);
+  private Integer getUsersCountInOktaGroup(String groupName) {
```

Comment:
Pagination needed the total number of users in an organization, so I split this logic out to be shared for facilities and organizations.